go/vt/sqlparser/normalizer_test.go

```diff
@@ -282,6 +303,14 @@ func TestNormalize(t *testing.T) {
 		outbv: map[string]*querypb.BindVariable{
 			"bv1": sqltypes.ValueBindVariable(sqltypes.MakeTrusted(sqltypes.Datetime, []byte("2022-08-06 17:05:12"))),
 		},
+	}, {
+		// TimestampVal should also be normalized
```

Comment:
Similar here, to ensure additional test coverage?

Reply:
Correct